editor/src/clj/editor/game_object.clj

```diff
@@ -546,7 +546,8 @@
   (let [self (:_node-id user-data)
         project (project/get-project self)
         component-type (:resource-type user-data)
-        template (workspace/template component-type)
+        workspace (:workspace user-data) ; Is there a better way to get it?
```

Comment:
I think you can get it from `(g/node-value project :workspace)` if you prefer to not pass it as user-data.